packages/react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -722,7 +802,7 @@ ComboBox.propTypes = {
    * Provide helper text that is used alongside the control label for
    * additional help
    */
-  helperText: PropTypes.string,
+  helperText: PropTypes.node,
```

Comment:
Shouldn't we change this to `string` as it was before?

Reply:
I think this is okay https://github.com/carbon-design-system/carbon/pull/14576#discussion_r1322003055